recipes/libnvpl-scalapack/meta.yaml

```diff
@@ -0,0 +1,142 @@
+
+
+{% set version = "0.2.0" %}
+{% set soversion = ".".join(version.split(".")[:3]) %}
+{% set somajor = version.split(".")[0] %}
+{% set arm_variant_type = arm_variant_type|default("sbsa") %}
+
+package:
+  name: libnvpl-scalapack-split
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/nvpl/redist/nvpl_scalapack/linux-sbsa/nvpl_scalapack-linux-sbsa-{{ version }}-archive.tar.xz
+  sha256: be7575647a9d192a2dd9bb908316da6deb422c71032880a4bb12d9e568c30b64
+
+build:
+  number: 0
+  # nvpl is only for ARM architecture
+  skip: true  # [not aarch64]
+  script:
+    - cp -rv include $PREFIX
+    - cp -rv lib $PREFIX
+
+outputs:
+
+  - name: libnvpl-scalapack-dev
+    build:
+      run_exports:
+        - {{ pin_subpackage("libnvpl-scalapack" ~ somajor ) }}
+    files:
+      - include/nvpl_scalapack*
+      - include/nvpl_scalapack*/**
+      - lib/cmake/nvpl_scalapack*/**
+      - lib/libnvpl_scalapack*.so
+      - lib/libnvpl_blacs*.so
+    requirements:
+      host:
+        - {{ pin_subpackage("libnvpl-scalapack" ~ somajor, exact=True) }}
+        - _nvpl_dev_mutex
+        - libnvpl-common-dev
+        - libnvpl-blas-dev
+        - libnvpl-lapack-dev
```

Comment:
@molah, the docs say that scalpack depends on blas and lapack, but there are no links detected by conda. I think the only dependency may actually be in the cmake files?